### PR TITLE
added healthcheck to config struct and healthcheck struct

### PR DESCRIPTION
**Pull request by chrisz100**
Needed to implement HEALTHCHECK commands to kaniko - see https://github.com/GoogleContainerTools/kaniko/issues/17#issuecomment-388868094

### CODE REVIEW

**Comment by chrisz100** [comment]
@dlorenc cc

File: v1/config.go [hunk 1]
```diff
@@ -64,6 +64,27 @@ type RootFS struct {
...
+// HealthConfig holds configuration settings for the HEALTHCHECK feature.
+type HealthConfig struct {
+	// Test is the test to perform to check that the container is healthy.
+	// An empty slice means to inherit the default.
+	// The options are:
+	// {} : inherit healthcheck
+	// {"NONE"} : disable healthcheck
+	// {"CMD", args...} : exec arguments directly
+	// {"CMD-SHELL", command} : run command with system's default shell
+	Test []string `json:",omitempty"`
```
Comment (dlorenc) [inline v1/config.go]: nit: should these be lowercased like `json:"test,omitempty"`

Comment (dlorenc) [reply v1/config.go]: Ah, basically just to make it lowercase when marshalled, to match the way we marshal all the other values in this file

**Comment by dlorenc** [comment]
Ah sorry for the back and forth, my coworkers point out that we actually should leave it the way you had it before to match exactly how Docker serializes. After that, this LGTM.

**Pull request merged**